command/operator_debug.go

```diff
@@ -593,16 +609,22 @@ func (c *OperatorDebugCommand) collectPprof(path, id string, client *api.Client)
 	bs, err := client.Agent().CPUProfile(opts, nil)
 	if err == nil {
 		c.writeBytes(path, "profile.prof", bs)
+	} else {
+		c.Ui.Error(fmt.Sprintf("%s: Failed to retrieve pprof profile.prof, err: %v", path, err))
```

Comment:
A couple of small issues with the error handling here:
* This error message is referring to a filepath, but the error is from the API call where we tried to construct that file yet.
* The handling of the error from `writeBytes` is inconsistent: we send _some_ of its errors to `c.Ui.Error` in that function but we also return them. But then they get thrown away here. So for example, if we fail to open the file that gets reported to the user, but if we fail to write to that file the error is being silently dropped.

Reply:
Agreed - I'm in the process of refactoring error handling as a whole as part of a separate PR.  This was a stop-gap solution to address issues we've encountered when a missing ACL/token is preventing pprof capture.